data/i-bmp-mappings.js

```diff
@@ -2174,12 +2178,18 @@ module.exports = new Map([
 	[0xA7C8, 0xA7C7],
 	[0xA7C9, 0xA7CA],
 	[0xA7CA, 0xA7C9],
+	[0xA7CB, 0x264],
+	[0xA7CC, 0xA7CD],
+	[0xA7CD, 0xA7CC],
 	[0xA7D0, 0xA7D1],
 	[0xA7D1, 0xA7D0],
 	[0xA7D6, 0xA7D7],
 	[0xA7D7, 0xA7D6],
 	[0xA7D8, 0xA7D9],
 	[0xA7D9, 0xA7D8],
+	[0xA7DA, 0xA7DB],
+	[0xA7DB, 0xA7DA],
+	[0xA7DC, 0x19B],
```

Comment:
The changes in `i-bmp-mappings.js` is only due to Unicode 16, where `1C89..1C8A`, `A7CB..A7CD` and `A7DA..A7DC` are introduced: https://www.unicode.org/Public/UCD/latest/ucd/DerivedAge.txt